tsconfig.json

```diff
@@ -6,5 +6,8 @@
 	},
 	"files": [
 		"index.ts"
-	]
+	],
+	"ts-node": {
+		"transpileOnly": true
```

Comment:
This fixes the tests in <=18, but Node 20 tests still broken even with the full

```json
		"transpileOnly": true
		"files": true,
		"experimentalResolver": true
```

That's why I had initially excluded them